index.bs

```diff
@@ -2401,14 +2405,31 @@ the {{ReadableStream}}'s public API.
  id="readable-stream-has-default-reader">ReadableStreamHasDefaultReader(|stream|)</dfn> performs the
  following steps:
 
- 1. Let |reader| be |stream|.\[[reader]].
+ 1. Let |reader| be |stream|.[=ReadableStream/[[reader]]=].
  1. If |reader| is undefined, return false.
  1. If |reader| [=implements=] {{ReadableStreamDefaultReader}}, return true.
  1. Return false.
 </div>
 
 <h4 id="rs-reader-abstract-ops">Readers</h4>
 
+Instances of {{ReadableStreamDefaultReader}} and {{ReadableStreamBYOBReader}} are created with the
+internal slots described in the following table:
```

Comment:
Is it weird to talk about "instances are created with the internal slots described in the following table" here, when actually they also have additional slots (i.e. `[[readRequests]]` and `[[readIntoRequests]]`) defined by the specific class as well?

Reply:
You can just expand it a bit. E.g. "In addition to the ones described in their individual sections, instances..."